klar_eda/visualize/image_visualize.py

```diff
@@ -25,7 +25,15 @@
 
 class ImageDataVisualize:
 
+
     def __init__(self, data, labels, boxes=None):
+        """Constructor for Image Data Visualization.

```

Comment:
Remove this file as well from the PR, to avoid conflicts. Can you please confirm if you have successfully rebased and taken the latest pull?

Reply:
I had taken the latest pool, but maybe I had made some minor changes while testing the code, which might've caused the conflicts. Resolving them nw.

Reply:
Can you please guide me on how to remove this file from PR

Reply:
Try using following commands
`git checkout HEAD^ -- /path/to/file`
`git commit -m 'Message'`
`git push`

Reply:
I tried this, it's showing the message
`nothing to commit, working tree clean`

Reply:
Let us move the conversation to slack

Reply:
Slack channel pls?

Reply:
['Here'](https://join.slack.com/t/klareda/shared_invite/zt-ndytbmnq-ARgpyJLPMbeFyYo87jrTjA)